### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by Gunter-B** [Courseplay/Courseplay_FS22#2827] [Q&A]
Hallo an alle, ich habe die Coursplay Version 7.3.0.0 auf 7.3.1.2 aktualisiert aber angezeigt wird mir immer die Version 7.3.0.0 ist das korrekt so ?  Auch die Funktion Silo Laden ist nicht verfügbar es wird immer nur Bunkersilo Modus angezeigt egal welches Werkzeug am Radlader verbaut ist. Danke schon mal für die Infos.

**Comment by Tensuko** [comment]
Die aktuelle Version bekommst du hier:
https://github.com/Courseplay/Courseplay_FS22/releases

Im Siel darfst du dann kein Update mehr durchführen, ansonsten wird deine Version auf die im ModHub zurück gesetzt.

Giants prüft nicht die Version, sondern den Hash-Wert der .zip Datei, somit kann auch eine ur alte version im ModHub als Update angezegit weren.

**Reply by Gunter-B** [reply]
Dankeschön für die schnelle Hilfe es hat funtioniert.
Danke nochmal.
